ExtensionHandler/Windows/src/net6.json

```diff
@@ -0,0 +1,137 @@
+[
```

Comment:
Isn't it a bit weird to have all these linux distros as supported in the windows handler?
Should this file be somewhere else and just once in the repo?

Reply:
We want this file to be present in the published zip folder for both Linux and Windows extension. The files present in both Linux and Windows folder are exactly same as we want these files to be consistent with the file present [here](https://raw.githubusercontent.com/microsoft/azure-pipelines-agent/master/src/Agent.Listener/net6.json). We were thinking of creating an automation that raises PR in this repo whenever the corresponding file is updated in the Agents repo @kirill-ivlev @tkasparek